### PR TITLE
home page of tes

### PR DESCRIPTION
**Pull request by mangalvikas161**
1) These is still a problem with splash screen other behind are also visible.
2) Call and email logo are not proper, as  I wasn't able to find the exact logo.
3) I have made a separate folder for images which includes all the logos used.
4) There are still some big issues of rem, em, vw, vh which on trying to see little bit responsive creating lot of changes.

### CODE REVIEW

File: public/sass/layouts/_home.scss [hunk 1]
```diff
@@ -0,0 +1,268 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
```
Comment (PraveenGupta007) [inline public/sass/layouts/_home.scss]: This isn't needed as we have already defined it in _base.scss

File: views/layouts/home.ejs [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<nav class="navbar-home">
+    <div class="navbar-home-title">tes</div>
+    <div class="menu_btn">
+        <div class="menu_btn-burger_icon"></div>
+    </div>
+</nav>
+
```
Comment (PraveenGupta007) [inline views/layouts/home.ejs]: Move this to header.ejs

File: views/layouts/home.ejs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+<div class="home-header">
+
+    <div>Textile</div>
+    <div>Enginneering</div>
+    <div>Society</div>
```
Comment (PraveenGupta007) [inline views/layouts/home.ejs]: Write this using a single div element

File: views/layouts/home.ejs [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            <div class="footer-home-left-footer-call-email-text">
+                <ul class=>
+                    <li><a href="#" style="text-decoration: none;">+91 9999999999</a></li>
+                    <li><a href="#" style="text-decoration: none;">tesiitd@gmail.com</a></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+
+</footer>
```
Comment (PraveenGupta007) [inline views/layouts/home.ejs]: Move this to footer.ejs

File: public/sass/layouts/_home.scss [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+
+.navbar-home{
+    background-color:  #D2D2D2;
+    position: absolute;
+    width: 100vw;
+    height: 6vh;
+    top: 0;
+
+    &-title{
+        float: left;
```
Comment (PraveenGupta007) [inline public/sass/layouts/_home.scss]: Don't use float. Do this using other properties.

**Comment by ssaify** [comment]
@PraveenGupta007  Whenever you will merge this PR please rebase and merge.

**Pull request merged**